completion/test_utils.py

```diff
@@ -65,7 +66,7 @@ def override_waffle_switch(self, override):
         Parameters:
             override (bool): True if tracking should be enabled.
         """
-        _waffle_overrider = waffle.waffle().override(waffle.ENABLE_COMPLETION_TRACKING, override)
+        _waffle_overrider = override_waffle_switch(waffle.waffle_switch(waffle.ENABLE_COMPLETION_TRACKING), override)
```

Comment:
This is unfortunately not working, as we remove the `.override` methods from the waffle namespace classes in edx-platform. edx-platform tests themselves depend on edx-completion's test_utils, so tests will not pass until we merge this.

Reply:
1. I presume you mean in this PR: https://github.com/edx/edx-platform/pull/25417/files?
2. In the edx-platform, I'd rather deprecate and then remove.  I usually add something like `set_custom_attribute('deprecated_THING_TO_BE_DEPRECATED', True)` to the deprecated methods so I can track that there really is no usage before removal.

We can still land this, but it might be better to land it after, rather than before, so I can see that the `deprecated_X` custom attribute was firing and finally stops. The main point is to see if anything else is using it that we didn't notice before removal.

Reply:
Yes, this is linked to https://github.com/edx/edx-platform/pull/25417.

The override functions are test methods, so my thinking was that it would be overkill to go through the deprecation process for them. But sure, I can do that.

> We can still land this, but it might be better to land it after, rather than before, so I can see that the deprecated_X custom attribute was firing and finally stops. The main point is to see if anything else is using it that we didn't notice before removal.

This needs to be merged before https://github.com/edx/edx-platform/pull/25417, or unit tests will fail in edx-platform -- unless we go through a deprecation process.